### PR TITLE
Clean-up Python #4815 - P1.9: Chem\Fraggle

### PR DESCRIPTION
**Pull request by IchiruTake**
This is the sub-PR breaking from the large PR 4815: https://github.com/rdkit/rdkit/pull/4815

### CODE REVIEW

File: rdkit/Chem/Fraggle/FraggleSim.py [hunk 1]
```diff
@@ -135,11 +132,11 @@ def select_fragments(fragments, ftype, hac):
       # needs to be greater 3 heavy atoms and greater than 40% of parent mol
       if isValidRingCut(fMol):
         result_hcount = fMol.GetNumAtoms()
-        if (result_hcount > 3) and (result_hcount > 0.4 * hac):
-          result = f
+        if result_hcount > 3 and result_hcount > 0.4 * hac:
+            result = f
```
Comment (gedeck) [inline rdkit/Chem/Fraggle/FraggleSim.py:136]: This looks like the indentation is now 4 spaces

File: rdkit/Chem/Fraggle/FraggleSim.py [hunk 2]
```diff
@@ -165,13 +162,11 @@ def select_fragments(fragments, ftype, hac):
...
-      result = '.'.join(result)
-    else:
-      result = None
-    return result
+      return '.'.join(result)
+    return None
 
-  else:
-    raise NotImplementedError('Invalid fragmentation type {0}'.format(ftype))
+  # This should never happen
```
Comment (gedeck) [inline rdkit/Chem/Fraggle/FraggleSim.py:168]: The comment is redundant and potentially misleading. NotImplementedError says that an implementation is missing and would need to be added

File: rdkit/Chem/Fraggle/FraggleSim.py [hunk 3]
```diff
@@ -277,7 +272,7 @@ def partialSimilarity(atomID):
   # generate mol object & fp for input mol (we are interested in the bits each atom sets)
   pMol = Chem.Mol(mol)
   aBits = []
-  _ = Chem.RDKFingerprint(pMol, atomBits=aBits, **rdkitFpParams)
+  Chem.RDKFingerprint(pMol, atomBits=aBits, **rdkitFpParams)
```
Comment (gedeck) [inline rdkit/Chem/Fraggle/FraggleSim.py:275]: See Greg's comment in previous PR. He prefers to keep the assignment to underscore to indicate that ignoring the result is on purpose

**Review by gedeck — CHANGES_REQUESTED** [review]
Small cosmetic changes

File: rdkit/Chem/Fraggle/FraggleSim.py (hunk 2 above)
Comment (IchiruTake) [reply rdkit/Chem/Fraggle/FraggleSim.py:168]: OK, I got your idea! I would fix it in the next patch. Thank you for your review.

File: rdkit/Chem/Fraggle/FraggleSim.py (hunk 3 above)
Comment (IchiruTake) [reply rdkit/Chem/Fraggle/FraggleSim.py:275]: OK, I got your idea! I would fix it in the next patch. Thank you for your review.

File: rdkit/Chem/Fraggle/FraggleSim.py (hunk 1 above)
Comment (IchiruTake) [reply rdkit/Chem/Fraggle/FraggleSim.py:136]: OK, I got your idea! I would fix it in the next patch. Thank you for your review.

**Review by gedeck — APPROVED** [review]

**Pull request merged**